inductiva/_cli/tasks.py

```diff
@@ -1,22 +1,71 @@
 """Tasks CLI subcommands."""
 from inductiva import tasks, utils
 from inductiva import _cli
+from inductiva.utils import format_utils
+from typing import Tuple
+from collections import defaultdict
 
 
-def list_tasks(args):
-    """List tasks."""
+def get_task_generic_info(list_of_tasks: list) -> Tuple[list, list]:
 
-    override_col_space = {
-        "Submitted": 18,
-        "Started": 18,
-        "Status": 10,
-        "Computation Time": 20,
-        "Resource Type": 20
+    table = defaultdict(list)
+
+    for task in list_of_tasks:
+        info = task.get_info()
+        status = task.get_status()
+
+        computation_end_time = info.get("computation_end_time", None)
+
+        execution_time = task.get_computation_time(fail_if_running=False)
+        if execution_time is not None:
+            if computation_end_time is None:
```

Comment:
Can't we remove this if. It seems natural that `if computation_end_time is None` then `status in ["started", "submited"]`.

Also general question: What is happening here for tasks that failed? Do we give the running time as well?

Reply:
Assuming that nothing weird happens with the status, If we have started or submitted the status, we can assume that we have no computation end time.

@IvanPombo do you think it's safe to remove this check? Or is this here for some reason?

If a task is killed, we give Computation Time = n/a.